### PR TITLE
Update sidebar to expose rules/payees as pages rather than modals

### PR DESCRIPTION
**Pull request by j-f1**
This makes it easier to access these tools and improves visibility.

<details><summary>[outdated screenshots]</summary>

Collapsed/Expanded:
 Screenshot_2022-08-31 17 15 24

Pages:
Screenshot_2022-08-31 17 16 00

</details>

### CODE REVIEW

**Comment by jlongster** [comment]
I like this direction, but something about schedules makes me think differently about it vs rules. In fact I think they are all a little different:

* Schedules is a unique feature that should feel first-class to the app. In my opinion it should stay top-level in the sidebar and shouldn't be treated as a "tool" hidden away
* Payees are entities created from transaction, and aren't typically something you interact with directly except for selecting one from a transaction. This probably maps the best to what you're trying to achieve with "tools"
* Rules are pretty low-level. imo they should be a bit more hidden away; I don't want users' first impression to think that they need to learn a complex rules system to use the app.

It was also intentional for the "accounts" link to sit above the account listing, making it feel like a header. However I'm OK not keeping that.

What do you think about a "More tools" link in the sidebar that opens a Menu to show a list of more tools? This would also make it easier to add a lot more stuff in the future. We could keep Schedules as a first-class item, and put the rest behind that menu. It would put less focus on it.

Actually, in the web version we kinda already have that; if you click the 3-dot menu in the top left of the app it opens that kind of menu. It's always been a bit weird to put "Manage Rules" etc in that menu though. We could take those out and put it behind a "More tools" menu that can be opened underneath the current links in the sidebar.

That would change the behavior where rules/payees are opened as pages instead of modals, but I think that's fine.

**Comment by j-f1** [comment]
Updates:


Screenshot_2022-09-08 12 32 40
Screenshot_2022-09-08 12 35 33

**Comment by j-f1** [comment]
Is there anything else I should be doing here?

**Comment by trevdor** [comment]
Hey @j-f1!, we're getting the ball rolling on these old PRs. 
If you're still around, mind fixing up the merge conflicts? Other than that, we're ready to approve it as-is.

File: packages/desktop-client/src/components/accounts/TransactionList.js [hunk 1]
```diff
@@ -160,6 +161,14 @@ export default function TransactionList({
     return newTransaction;
   }, []);
 
+  let onManagePayees = useCallback(
+    id => {
+      debugger;
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/TransactionList.js:166]: **🔨 warning:** ‏need to remove this
```suggestion
```

File: packages/desktop-client/src/components/modals/ManageRulesModal.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+
+import { Modal } from 'loot-design/src/components/common';
+
+import ManageRules from '../ManageRules';
+
+export default function ManageRulesModal({ modalProps, payeeId }) {
+  let [loading, setLoading] = useState(true);
+  if (process.env.NODE_ENV === 'development') {
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/modals/ManageRulesModal.js:9]: **💬 suggestion:** ‏would you mind using `isDevelopmentEnvironment()` util function here?

**Review by MatissJanis — COMMENTED** [review]
Thanks for the contribution!

One nitpick and one bigger issue (`debugger`). Once you solve the debugger issue I'm happy to immediately merge this.

**Comment by j-f1** [comment]
Done!

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**